### ISSUE TITLE
Summarizing key takeaways from review of past research

### ISSUE THREAD

**Issue by Ndoyle118** [hackforla/expunge-assist#600]
### Overview
Conduct review of existing user personas, journey maps, and stakeholder interviews on the EA Shared Drive to help align new and current research team members and better understand insights gained from past research as well as plan for upcoming research. This issue is dependency for #473 

### Action Items - User Personas and Journey Maps
- [x] Review User Personas and journey maps on EA Shared Drive (Nick)
- [x] Discuss highlights with research team 
- [x] Create [1-page document](https://docs.google.com/document/d/1YJByabmc-PAhvM2khlwy4jrzJFGoYIXmn4NgniQRvhg/edit) to summarize key takeaways
- [x] Upload document to shared drive

### Action Items -Stakeholder Interviews
- [x] Review Stakeholder Interviews on EA Shared Drive (Nikita)
- [x] Discuss highlights with research team 
- [x] Create [1-page document](https://docs.google.com/document/d/1YJByabmc-PAhvM2khlwy4jrzJFGoYIXmn4NgniQRvhg/edit) to summarize key takeaways
- [x] Upload document to shared drive

### Resources/Instructions

- CfA User Personas: https://drive.google.com/drive/u/0/folders/12-1yK0UMW_XH1Q6sTtlXmq0LjVu7E_Tr
- Service Design Map Miro Board: https://docs.google.com/document/d/18BZV43VYXuPMofNgFK6uKxR_sXP1A1v6/edit
- Stakeholder Meeting Notes: https://docs.google.com/document/d/1wfcTstpRzcH0wn9MvIb7H7eCF30XDHf20-6SpZpG29A/edit#heading=h.prsheldg1dhj

**Comment by ValeriyaMetla** [comment]
@Ndoyle118 can you add size and feature label. Also, attach relevant documents like links to summaries pages and any major documents you reviewed for user personas, journey maps, and stakeholder interviews. Otherwise looks great!

**Comment by Ndoyle118** [comment]
@nikitashah14 Can you take a look at the 'Stakeholder Meeting Notes' document attached in the issue and let me know if this is what you looked at, and if there were any other pages you reviewed when looking at Stakeholder Interviews, feel free to add them!

**Comment by nikitashah14** [comment]
Yes, that is the correct document. I also checked out the Interview Guide, which has an overview and script from the Stakeholder Interviews. I added the main takeaways from what we had discussed during our team meeting, but feel free to add anything else in case I missed other points in the User Personas/Journey Maps section! 

[](https://docs.google.com/document/d/1MmgMMEtltrPqtWNrhXVgmEBveT_yqdXZwcA03z2KNzc/edit#heading=h.96tv3otdl0ds)
[](https://docs.google.com/document/d/1YJByabmc-PAhvM2khlwy4jrzJFGoYIXmn4NgniQRvhg/edit)
